tools/sup-network/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:18.04
+
+RUN apt-get update
+
+RUN apt-get install -y \
+            libarchive-dev \
+            libczmq-dev \
+            libsodium-dev \
+            libssl-dev
```

Comment:
Is it possible to access what we need there through habitat packages as we do in https://github.com/habitat-sh/habitat/blob/master/test/run_cargo_test.sh#L29-L52 ?

Reply:
I'm guessing that won't work because I'm purposely not installing `hab` into this container.  If I were to install `hab`, it would conflict with the bind mount of `/bin/hab` to `habitat/target/debug/hab` on the host machine.